modules/card.js

```diff
@@ -223,6 +230,9 @@ class MtgCardLoader {
             }
 
             let description = this.generateDescriptionText(card);
+            if (command.match(/^art/)) {
+                description = '🖌️' + card.artist
```

Comment:
I think a space between the icon and artist would be good here, but I'll leave that up to @bra1n

Reply:
honestly, you can't really tell the difference in a non-monospaced font, but I can make the change

Reply:
Agreed!